pkg/policies/policies.go

```diff
@@ -261,22 +256,33 @@ func (pv *PolicyVerifier) getLoader(attachment *v1.PolicyAttachment) (Loader, er
 	return loader, nil
 }
 
-func validatePolicy(policy *v1.Policy) error {
+func validateResource(m proto.Message) error {
 	validator, err := protovalidate.New()
 	if err != nil {
 		return fmt.Errorf("validating policy spec: %w", err)
 	}
-	err = validator.Validate(policy)
+	err = validator.Validate(m)
 	if err != nil {
 		return fmt.Errorf("validating policy spec: %w", err)
 	}
 
 	return nil
 }
 
-func getInputArguments(inputs map[string]string) map[string]any {
+func getInputArguments(inputs map[string]string, overrides map[string]string) map[string]any {
 	args := make(map[string]any)
-	for k, v := range inputs {
+
+	allInputs := make(map[string]string)
+	// Override inputs into a new map
+	maps.Copy(allInputs, inputs)
+	maps.Copy(allInputs, overrides)
+
+	for k, v := range allInputs {
+		// look for override
+		ov := overrides[k]
```

Comment:
I find this override logic confusing since it prevents us from having policies in a group with the same argument if they mean something different.

The way I would have thought of doing this is by actually letting the user explicitly setting the value similarly to github actions 

something like

```
- ref: cyclonedx-freshness
        with:
          freshness: {{ group.inputs.freshness }}
```

It feels way more controlled that way and you can see from within the group that you allow overrides, you might not want that

Reply:
Sorry, it has old logic, that's why it's confusing. In any case, I'll remove the whole override logic to have a broad discussion about policy inputs in general.